content/applications/websites/website/web_design/building_blocks.rst

```diff
@@ -0,0 +1,144 @@
+
+Grid
+----
+
+The :guilabel:`Grid` layout allows you to reposition and resize elements, such as images or text, by
+dragging and dropping them.
+
+.. image:: building_blocks/grid-layout.png
+   :alt: When the grid layout is selected, choose an image and drag and drop it where needed.
+
+.. tip::
+   Position images behind the text by using the above/below icons.
+
+   .. image:: building_blocks/superimpose-images-to-text.png
+      :alt: Use the above/below icons to superimpose images to text.
+
+Cols
+----
+
+Choosing the :guilabel:`Cols` layout allows you to determine the number of elements per line within
+the block.
+
+.. image:: building_blocks/elements-per-line.png
+   :alt: Set the number of elements per line in the block.
+
+.. note::
+   On mobile devices, only one element per column is used by default for better visibility. You can
+   adjust this value by using the :guilabel:`Mobile Preview` mode.
+
+Duplicating a building block
+============================
+
+You can duplicate a building block by clicking on the duplicate icon. Once duplicated, the new block
+appears on your website beneath the original one.
+
+.. image:: building_blocks/duplicate-container.png
+   :alt: Replicating a building block
+
+Reordering building blocks
+==========================
+
+To reorder the building blocks, select the block you want to move and click the up arrow to move it
+before the previous section, or click the down arrow to move it after the following section.
+
+You can also use the drag-and-drop icon to move your block manually.
+
+.. image:: building_blocks/re-ordering-blocks.png
+   :alt: Reorder building blocks by using the up and down arrow icons.
+
+Visibility
+==========
+
+Visibility on mobile
+--------------------
+
+You can hide specific elements of your website for mobile devices. To do so, select the element
+to hide, and in the :guilabel:`Customize` tab, scroll down to :guilabel:`Visibility`, and click the
+:guilabel:`Show/Hide on Mobile` or the :guilabel:`Show/Hide on Desktop` icon.
+
+.. image:: building_blocks/show-hide-on-mobile.png
+   :alt: Click the "show/hide on mobile" icons to show or hide some elements on mobile.
+
+.. tip::
+   Click the phone icon at the top of the configurator to preview your website appearance on a
+   mobile device.
+
+   .. image:: building_blocks/phone-icon.png
+      :alt: Click the phone icon to view how your website renders on mobile.
+
+Conditional visibility
+----------------------
+
+You can also conditionally render visible or hidden for certain building blocks. To do so, select
+the element you want to show/hide, go to :guilabel:`Visibility`, and click :guilabel:`Conditionally`.
+Then, choose the conditions to apply based on the :guilabel:`Country` (based on Geo IP),
+:guilabel:`Languages` (for multi-language websites), :guilabel:`UTM Campaign`,
+:guilabel:`UTM Medium`, :guilabel:`UTM Source`, or :guilabel:`Users`. The options available depend
+on your database's settings.
```

Comment:
```suggestion
You can also hide or show building blocks using other conditions. To do so, select
an element, go to :guilabel:`Visibility`, click :guilabel:`No condition`, and select :guilabel:`Conditionally` instead.
Then, configure the condition(s) to apply by selecting :guilabel:`Visible for` or :guilabel:`Hidden for` and which  :guilabel:`Records` will be impacted.
```
- The first sentence isn't clear.
- By default, you don't see ***Conditionally*** but ***No condition***
- You don't choose conditions based on _xxx_ (a condition). For a condition, you have to choose: which records the condition applies to and if the block should be shown or hidden.